nemo/collections/llm/recipes/llama3_8b.py

```diff
@@ -92,6 +93,22 @@ def pretrain_recipe(
     )
 
 
+def pretrain_recipe_performance(
+    name: str, ckpt_dir: str, num_nodes: int, num_gpus_per_node: int, fn: Callable = pretrain
+) -> Partial:
+    recipe = pretrain_recipe(
+        name=name, ckpt_dir=ckpt_dir, num_nodes=num_nodes, num_gpus_per_node=num_gpus_per_node, fn=fn
+    )
+
+    recipe.trainer.callbacks.append(
+        Config(
+            MegatronCommOverlapCallback,
+            tp_comm_overlap=False,
```

Comment:
why is it false here ?

Reply:
See https://github.com/NVIDIA/NeMo/pull/10209#discussion_r1745017511
Maybe Im misunderstanding this?